source/management_and_operations/backups/operations.rst

```diff
@@ -11,18 +11,25 @@ Backup Types
 --------------------------------------------------------------------------------
 OpenNebula supports two backup types:
 
-- **Full**, each backup contains a full copy of the VM disks.
-- **Incremental**, each backup contains only the changes since the last backup. **Important**: Incremental backups are only available for KVM and Qcow2 disks.
+- **Full**, each backup contains a full copy of the VM disks. Libvirt version >= 5.5 is required.
+- **Incremental**, each backup contains only the changes since the last backup. Libvirt version >= 7.7 is required.
 
 The Backup Process
 --------------------------------------------------------------------------------
-VM backups can be taken live (only for KVM virtual machines) or while the VM is powered-off, the operation comprises three steps:
+VM backups can be taken live or while the VM is powered-off, the operation comprises three steps:
 
 - *Pre-backup*: Disks (or increments) are prepared for backup. When the VM is running the filesystems of the guest are frozen (see below) and temporal disks are created so the VM can continue its normal operation. Note: backups are taken at the same time for all the VM disks (qcow2/raw images) to guarantee **crash consistent backups**.
 - *Backup*: Full disk copies (or increments) are uploaded to the backup server. In this step, OpenNebula will use the specific datastore drivers for the backup system.
 - *Post-backup*: Cleans any temporal file in the hypervisor.
 
-**Note**: In order to save space in the backup system, disk backups are stored always in Qcow2 format.
+.. note:: In order to save space in the backup system, disk backups are stored always in Qcow2 format.
+
+Limitations
+============
+- Incremental backups are only available for KVM and qcow2 disks
+- Live backups are only supported for KVM
+- Attaching a disk to a VM that had an incremental backup previosly made will yield an error on the next backup
```

Comment:
Add here that you need to do a --reset operation so a new incremental chain is created.